cmd/dagger/module.go

```diff
@@ -1116,6 +1116,7 @@ var skipLeaves = map[string][]string{
 		// stdout and stderr may be arbitrarily large and jarring to see (e.g. test suites)
 		"stdout",
 		"stderr",
+		"exitCode",
```

Comment:
I think you can leave `exitCode` out since it doesn't trigger execution anymore. We only left `stdout` and `stderr` anyway due to potentially large output.

Reply:
:tada: added a comment.